sdks/python/apache_beam/coders/typecoders_test.py

```diff
@@ -33,14 +39,18 @@ def __init__(self, n):
   def __eq__(self, other):
     return self.number == other.number
 
+  def __hash__(self):
+    return self.number
+
 
 class CustomCoder(coders.Coder):
 
   def encode(self, value):
-    return str(value.number)
+    x = value.number
+    return int(x).to_bytes((x.bit_length() + 7) // 8, 'big', signed=True)
```

Comment:
<!--new_thread; commit:e3d8e1e2f5f597ff6bf0e896953f3996e74d8022; resolved:0-->
What is the rationale for changing the coder behavior here to use a binary encoding instead of using a printable decimal string, for what is intended to be a simple example?  Is there something simpler we can do? (e.g. we could coerce to `str` and encode as `.encode('latin-1')`)

Reply:
That would be simpler indeed. It clashed with the cython bytes, since `str().encode('latin-1')` will return `newbytes` with the future package. 
However, seems like we need to fix this cython bytes issue anyway, so I have changed it back. This is the drawback of working on a per module basis I guess :)